### ISSUE TITLE
env variable in container_spec of docker_service resource

### ISSUE THREAD

**Issue by FortuneLenovo** [terraform-providers/terraform-provider-docker#74]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

### Affected Resource(s)
Please list the resources as a list, for example:
- opc_instance
- opc_storage_volume

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by FortuneLenovo** [comment]
Hi Mavogel,

I can see you formatted the syntax, actually i did already tried with correct format. Pasted here without format was just for your reference.
Please help me to find out why i am getting thee blank environment parameters.

Thanks,

**Comment by FortuneLenovo** [comment]
Hi,

While configuring service inside docker_service resource, i want to attach network with desired IP address while in your guide i am able to find to attach with network only

networks     = ["${docker_network.test_network.id}"]

this is there, please suggest how can i choose my desired IP while attaching network.

Thanks in advance.

**Comment by mavogel** [comment]
Will investigate your last comment together with #36. The initial issue was solved in #81
